### PR TITLE
Support solution removal through solution-manager

### LINKED ISSUE #1596: Add the possibility to remove a solution through solution manager
**Component**:
solutions

**Why this is needed**:
until now the solution-manager script only adds new solutions, it will be useful to have solution deletion supported
**What should be done**:

**Implementation proposal** (strongly recommended):

**Test plan**:

### PR DESCRIPTION
**Pull request by sayf-eddine-scality**
Fixes: #1596

**Component**:

**Context**: 

**Summary**:

**Acceptance criteria**: 

---

Closes: #1596

### CODE REVIEW

File: scripts/solution-manager.sh [hunk 1]
```diff
@@ -16,16 +16,23 @@ PILLAR=""
 _usage() {
     echo "solution-manager.sh [options]"
     echo "Options:"
-    echo "-s/--solution <solution path>:   Path to solution ISO"
+    echo "-a/--add <solution path>:        Add solution from ISO path"
+    echo "-d/--del <solution path>:        uninstall solution from ISO path"
```
Comment (slaperche-scality) [inline scripts/solution-manager.sh]: Uninstall (use uppercase, to be consistent with the other strings)

File: scripts/solution-manager.sh [hunk 2]
```diff
@@ -16,16 +16,23 @@ PILLAR=""
...
     echo "-l/--log-file <logfile_path>:    Path to log file"
     echo "-v/--verbose:                    Run in verbose mode"
 }
 
-SOLUTIONS=()
+SOLUTIONS_ADD=()
+SOLUTIONS_REMOVE=()
+existent_solutions=()
```
Comment (slaperche-scality) [inline scripts/solution-manager.sh]: Why this one is not in uppercase?

File: scripts/solution-manager.sh [hunk 3]
```diff
@@ -175,29 +182,52 @@ containsElement () {
...
-        fi
-    done
+    # Add new solutions
+    if [ ${#SOLUTIONS_ADD[@]} -ge 1 ]; then
+        for solution in "${SOLUTIONS_ADD[@]}"; do
+            if ! containsElement "'$solution'" \
+                 "${EXISTENT_SOLUTIONS[@]+"${EXISTENT_SOLUTIONS[@]}"}"; then
+                EXISTENT_SOLUTIONS+=("'$solution'")
+            fi
+        done
```
Comment (TeddyAndrieux) [inline scripts/solution-manager.sh]: To be consistent can you create the function `add_solutions`

Comment (sayf-eddine-scality) [reply scripts/solution-manager.sh]: does it make sense to have a function with just `EXISTENT_SOLUTIONS+=("'$solution'")` ?

Comment (TeddyAndrieux) [reply scripts/solution-manager.sh]: The function will also do the `for` loop + the `if ! containsElement`.
So you will have something like
```
if "SOLUTIONS_ADD not empty" then
  add_soltuions SOLUTIONS_ADD
if "SOLTUIONS_REMOVE not empty" then
  remove_soltuions SOLUTIONS_REMOVE
```
Which is cleaner IMO

**Review by TeddyAndrieux — APPROVED** [review]
LGTM

**Pull request merged**